ReactWindows/ReactNative.Tests/Modules/Clipboard/ClipboardModuleTests.cs

```diff
@@ -0,0 +1,115 @@
+﻿using Microsoft.VisualStudio.TestPlatform.UnitTestFramework;
+using ReactNative.Bridge;
+using ReactNative.Modules.Core;
+using ReactNative.Modules.Clipboard;
+using System;
+using System.Threading;
+
+namespace ReactNative.Tests.Modules.Clipboard
+{
+    [TestClass]
+    public class ClipboardModuleTests
+    {
+        [TestMethod]
+        public void ClipboardModule_ArgumentChecks()
+        {
+            var module = CreateClipboardModule(new MockInvocationHandler());
+
+            AssertEx.Throws<ArgumentNullException>(
+                () => module.getString(null),
+                ex => Assert.AreEqual("promise", ex.ParamName));
+        }
+
+        [TestMethod]
+        public void ClipboardModule_GetString_Method()
+        {
+            var module = CreateClipboardModule(new MockInvocationHandler());
+
+            var waitHandle = new AutoResetEvent(false);
+            Promise promise = new Promise(waitHandle);
+            string str = "test string";
+
+            module.setString(str);
+            module.getString(promise);
+
+            waitHandle.WaitOne();
+            Assert.AreEqual(str, promise.value);
+
+            module.setString(null);
+            module.getString(promise);
+
+            waitHandle.WaitOne();
+            Assert.AreEqual("", promise.value);
+        }
+
+        private static ClipboardModule CreateClipboardModule(IInvocationHandler handler)

```

Comment:
You won't need this parameter if you're not instantiating the RCTDeviceEventEmitter.